rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -2279,3 +2279,153 @@ rocalToDecibels(
     }
     return output;
 }
+
+RocalTensor ROCAL_API_CALL
+rocalResample(RocalContext p_context,
+              RocalTensor p_input,
+              RocalTensor p_input_resample_rate,
+              RocalTensorOutputType rocal_tensor_output_datatype,
+              bool is_output,
+              float sample_hint) {
+    if (!p_context || !p_input_resample_rate || !p_input)
+        THROW("Null values passed as input")
+    Tensor* resampled_output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Tensor*>(p_input);
+    auto input_resample_rate = static_cast<Tensor*>(p_input_resample_rate);
+    RocalTensorDataType op_tensor_data_type;
+    try {
+        TensorInfo output_info = input->info();
+        op_tensor_data_type = (RocalTensorDataType)rocal_tensor_output_datatype;
+        output_info.set_tensor_layout(RocalTensorlayout::NONE);
+        output_info.set_data_type(op_tensor_data_type);
+        if (sample_hint > 0) {
+            std::vector<size_t> max_dims = output_info.max_shape();
+            std::vector<size_t> dims = output_info.dims();
+            dims[1] = std::ceil(sample_hint);
+            dims[2] = max_dims[1];
```

Comment:
Instead of max_dims, Why not use output_info.dims()[2]?
Or else this dimension need not be changed right?